src/universal/containers/Logout/LogoutContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, {PropTypes, Component} from 'react';
+import {connect} from 'react-redux';
+import {push} from 'react-router-redux';
+import {show} from 'universal/modules/notifications/ducks/notifications';
+import {removeAuthToken} from 'universal/redux/authDuck';
+
+const logoutSuccess = {
+  title: 'Tootles!',
+  message: 'You\'ve been logged out successfully.',
+  level: 'success'
+};
+
+@connect()
+export default class LogoutContainer extends Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired
+  };
+
+  componentWillMount() {
+    const {dispatch} = this.props;
+    dispatch((thunkDispatch) => Promise.all([
```

Comment:
Follow along at https://github.com/michaelcontento/redux-storage/pull/148.
I'll give it a day or 2 before I change `package.json` to point to the version in my github repo.